CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Changelog
 
-## [2.3.0](https://launchpad.net/mixxx/+milestone/2.3.0)
+## 2.3.1
+* AutoDJCratesDAO: Fix crash after loading/unloading tracks. [lp1936396](https://bugs.launchpad.net/mixxx/ [#4107](https://github.com/mixxxdj/mixxx/pull/4107)+bug/1936396)
```

Comment:
This was a temporary issue of the *History* fix and should not be mentioned as a change. It didn't affect 2.3.0.

Reply:
this is a lot of work to update the changelog, we need to figure out a process to make this easier.